lib/types/host-rules.ts

```diff
@@ -3,6 +3,7 @@ export interface HostRuleSearchResult {
   token?: string;
   username?: string;
   password?: string;
+  sshKey?: string;
```

Comment:
we should probably also support an `sshKeyFile` property, like we do for `gpgKey`?

@rarkins WDYT?

Reply:
We could wait and see if someone needs it, it can easily be added later

Reply:
Not sure how easy it will be to implement. On server side (like `gpgKey`) its ok to have a key unencrypted. But in the repo config, we definitely want supported for encryption. `sshKey` automatically benefits from the `encrypted` config. But `sshKeyFile` would not. 

Therefore we either need to add `sshKeyFile` and `sshKeyFileEnc` to allow both or only `sshKeyFile` but then it must be encrypted. In both cases we need to extend the decryption logic to read the file and decrypt it.

Reply:
we'll defer this until someone wants it